src/main/java/org/example/homework7/RomanNumbersCalculator/RomanNumber.java

```diff
@@ -0,0 +1,13 @@
+package org.example.homework7.RomanNumbersCalculator;
+
+public enum RomanNumber {
+    I(1), IV(4), V(5), IX(9), X(10),
```

Comment:
better to write everything in one line or every number in a new line
```suggestion
    I(1), 
    IV(4),
    V(5),
    IX(9),
    X(10),
```

Reply:
Improved